edb/server/protocol/auth_ext/http.py

```diff
@@ -2534,15 +2540,6 @@ def _set_cookie(
     response.custom_headers["Set-Cookie"] = val.OutputString()
 
 
-def _with_appended_qs(url: str, query: dict[str, list[str]]) -> str:
```

Comment:
We had duplicate ways of adding to a URL's query string, so went with the more recent `util`-based function and removed this one. The only difference is that this one handles arrays of values, but we were not actually using that functionality.